### ISSUE TITLE
[Strategy Market][Switch]Base limits

### ISSUE THREAD

**Issue by vineetsk1** [vineetsk1/Switch-Strategies#12]
Original Author: salish99
Original Link: https://getsatisfaction.com/eternagame/topics/-strategy-market-switch-base-limits

Thsi may already have been implemented. 

In accordance with 
https://getsatisfaction.com/eternagam... 

1) To achieve high binding energies, we need: 
a) 10-15% C 
b) 35-60 % A 
c) 10-16 % U 
d) 35 % G for Ex3, and 12-16 % G for Ex4 and SS1 

and for the second data set 

2) To achieve high Eternascores, we need: 
a) 10-24 % C 
b) 25-58 % G 
c) 8-25 % U 
d) 18-40 % G 
! 
These spans are, unfortunately, so wide that they likely encompass 3/4 of all the molecules in this (or any other) data set. 

Of particular interest are those runaway data lines (e.g. at 38 % A and corresponding 14 % U for Ex3), where a whole set of very high binding energies can be achieved with essenstially the same composition, and just one minute change in the molecule makes a large difference, whereas any such change in molecules outside this very defined parameter set changes the binding energy barely at all. Similar peaks exist for other experiments, too, but I wanted to point this one out as it's very prevalent across the first four graphs.

**Comment by salish99** [comment]
remember that this is a typo from Feb 2015, the second point is C, A, U, G

**Comment by vineetsk1** [comment]
@salish99 yep, the post here is only your original post on the getsat forum - I will be implementing based on your latest reply.

**Comment by vineetsk1** [comment]
Implemented - 25 lines of code.
